CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Release 2.44.0 06-04-2023
+
+- Update Envoy to v1.25.4 (#809)
+- Bump API version v11 -> v12 (#808)
```

Comment:
remove, this is internal API for ESPv2 users

Reply:
We regard CAG as our customers as well and we have changelog for API version update before as well.

https://screenshot.googleplex.com/3jC4ghRxg5aApRu